src/services/api.ts

```diff
@@ -75,10 +77,10 @@ const config = new Configuration({
   basePath: AMP_API_ROOT,
 });
 
-let apiValue = new DefaultApi(config);
+let apiValue = new ApiService(config);
 
 // For testing, etc. we may want to use a different API configuration than the default
-export const setApi = (api: DefaultApi) => {
+export const setApi = (api: ApiService) => {
```

Comment:
This may or may not break tests if they are written with new service layer. @laurenzlong